dactyl-config.yml

```diff
@@ -597,11 +597,227 @@ pages:
+        html: wallet_propose.html
+        funnel: Docs
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Key Generation Methods
+        targets:
+          - local
+
+    -   name: Logging and Data Management Methods
+        html: logging-and-data-management-methods.html
+        funnel: Docs
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Logging and Data Management Methods
+        template: template-landing-children.html
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/can_delete.md
+        html: can_delete.html
+        funnel: Docs
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Logging and Data Management Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/ledger_cleaner.md
+        html: ledger_cleaner.html
+        funnel: Docs
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Logging and Data Management Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/ledger_request.md
+        html: ledger_request.html
+        funnel: Docs
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Logging and Data Management Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/log_level.md
+        html: log_level.html
+        funnel: Docs
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Logging and Data Management Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/logrotate.md
+        html: logrotate.html
+        funnel: Docs
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Logging and Data Management Methods
+        targets:
+          - local
+
+    -   name: Server Control Methods
+        html: server-and-control-methods.html
+        funnel: Docs
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Server Control Methods
+        template: template-landing-children.html
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/server-control-methods/connect.md
+        html: connect.html
+        funnel: docs
```

Comment:
This needs to be an uppercase "Docs"—it's causing weirdness in the sidebar.

You probably typed it correctly, but (annoyingly) Atom seems to want to autocorrect this to lowercase. So beware of that moving forward.

Reply:
Ohh, I was wondering what was causing that and couldn't figure it out!